### ISSUE TITLE
adding highway=service to every single leisure=slipway makes no sense

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#6966]
Is it intentional that iD wants to add

```
highway=service
service=slipway
```

to every single slipway, including ones mapped as areas where it break iD rendering of this feature?

See say https://en.wikipedia.org/wiki/Slipway#/media/File:Slip-V%C3%A4rldens_%C3%A4nde-Ystad.jpg or https://en.wikipedia.org/wiki/Slipway#/media/File:Slipway_at_portland.JPG

highway=service makes sense on many linear slipways (see case like https://en.wikipedia.org/wiki/Slipway#/media/File:Slipway.JPG ), or maybe even on most of slipways. But adding it to every single slipway is weird.

**Comment by quincylvania** [comment]
@matkoniecz iD doesn't support these as areas at all. The issue is that the same word/tag is used for several different classes of things. It'd be nice to have tags that differentiate one type of slipway from another, but I agree not all classes are appropriate for vehicles.

**Comment by dieterdreist** [comment]
yes, it is common that the same word can mean different things, in English but also in other languages. Ideally there should be some kind of second question to make sure the tags that are set are corresponding to the feature that the mapper wants to tag.
